KPI-team1-frontend/src/components/Sidebar.tsx

```diff
@@ -5,12 +5,31 @@ import { HiOutlineMagnifyingGlass } from "react-icons/hi2";
 import { HiStar } from "react-icons/hi2";
 import { AiOutlineSetting } from "react-icons/ai";
 import { FiLogIn } from "react-icons/fi";
+import { FiLogOut } from "react-icons/fi";
 import { useNavigate } from "react-router-dom";
+import { supabase } from "../supabase";
 
-export default function Sidebar(): JSX.Element {
+interface User {
```

Comment:
Since we are using this two times already (and probably we'll be using it more), let's extract this to a separate package. Maybe it could be called `model`?

Reply:
Will do that 👌🏻